crates/weaver_common/src/lib.rs

```diff
@@ -55,6 +58,7 @@ pub trait Logger {
 pub struct ConsoleLogger {
     logger: Arc<Mutex<paris::Logger<'static>>>,
     debug_level: u8,
+    mute: Arc<Mutex<bool>>,
```

Comment:
I'd just use an atomic here.

Reply:
I agree, a mutex is overkill in this context. An atomic is indeed enough, I don't know why I didn't use it in the first place.

Reply:
Replaced the mutex by an atomic in [d14e711](https://github.com/open-telemetry/weaver/pull/187/commits/d14e711c0028d10c21b260cbf287db549b6dc8d4)